nautobot/core/templates/rest_framework/api.html

```diff
@@ -27,27 +36,32 @@
         {% block header %}{% endblock header %}
     </header>
     <main class="container-fluid wrapper" id="main-content" tabindex="-1">
-        {{ block.super }}
+        {% block content %}
+            {{ block.super }}
+        {% endblock %}
+
+        {% if filter_form %}
+          {{ filter_form }}
```

Comment:
I *think* we're also hiding the filter_form in our views? Would be good to confirm though.